src/views/Sale/components/SelfBidList/index.tsx

```diff
@@ -146,7 +146,7 @@ export function SelfBidList({ sale, clearingPrice, bids, isFixed }: SelfBidListP
 
         {bids.map((bid: FixedPriceSalePurchase, index: number) => {
           const bidValue =
-            formatBigInt(sale.tokenPrice, sale.tokenOut.decimals) * formatBigInt(bid.amount, sale.tokenOut.decimals)
+            formatBigInt(sale.tokenPrice, sale.tokenIn.decimals) * formatBigInt(bid.amount, sale.tokenOut.decimals)
```

Comment:
Is this supposed to fix the issue we found the other day?

If so I think it is still broken. The below shows the UI but the amount I put in and withdrew out was equal to 1.1, which is not what the UI screenshot suggests. I think you might have gotten it the wrong way around.

Current (what the UI values, currently in this branch, add up to):
Totals- 1.1 FPST 0.55 DAI

Correct (what the values should show):
Totals- 2.2FPST 1.1 DAI

<img width="416" alt="2021-06-17 19 24 45" src="https://user-images.githubusercontent.com/39137239/122452915-ad5a6980-cfa1-11eb-8108-c95bc36e43cd.png">

Reply:
No, those changes were supposed to be stashed as it was later found out that in this [component](https://github.com/cryptonative-ch/mesa-interface/blob/24862705a26379c481ad69ead2b4270abef4438b/src/views/Sale/components/PurchaseTokensForm/index.tsx#L140), that it's being denominated in the input token rather than the amount being denominated in the Output token. The smart contracts [input amount](https://github.com/cryptonative-ch/mesa-smartcontracts/blob/26a59272a7014c41dbb88616b9fa0736ef1a94b8/contracts/sales/FixedPriceSale.sol#L171-L190) is denominated in  `tokenOut` as you can see that it directly makes a comparison between the user amount and the [`allocationMax` and `allocationMin`](https://github.com/cryptonative-ch/mesa-smartcontracts/blob/26a59272a7014c41dbb88616b9fa0736ef1a94b8/contracts/sales/FixedPriceSale.sol#L71-L72),